src/WebJobs.Script/Description/CSharp/CSharpFunctionInvoker.cs

```diff
@@ -66,13 +66,14 @@ public class CSharpFunctionInvoker : ScriptFunctionInvokerBase
         private static string GetTriggerInputName(FunctionMetadata functionMetadata)
         {
             BindingMetadata triggerBinding = functionMetadata.Bindings.FirstOrDefault(b => b.IsTrigger);
-            
+
+            string triggerName = null;
             if (triggerBinding != null)
             {
-                return triggerBinding.Name;
+                triggerName = triggerBinding.Name;
```

Comment:
That's correct. This will address an issue that was reported where we failed to process the response of an HTTP Trigger because the trigger name wasn't properly set (in order to locate the parameter containing the trigger's input)